src/main/java/nl/jessicavdz/redlightgreenlight/Block/BlockBreakCancel.java

```diff
@@ -0,0 +1,15 @@
+package nl.jessicavdz.redlightgreenlight.Block;
+
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockBreakEvent;
+
+import java.util.EventListener;
+
+public class BlockBreakCancel implements EventListener, Listener {
```

Comment:
Why does this implement java.util.EventListener?

Reply:
I've fixed it!